docs/release-notes/deprecate-modularity.rst

```diff
@@ -3,8 +3,9 @@
 
 :Description:
     Based on the DNF team recommendation we decided to deprecate RPM modularity
```

Comment:
I guess not `deprecate` but `remove`?

Reply:
Well - removal according to kickstart results in error: https://github.com/pykickstart/pykickstart/blob/master/CONTRIBUTING#L127

I dont know what the best wording here is. It's highly confusing - as deprecation indeed normally means plan to remove.

Reply:
I wouldn't recommend to take the pykickstart solution into account at all. It's confusing even for us and it shouldn't be propagated to release notes for external people.